package.json

```diff
@@ -2,50 +2,52 @@
   "name": "ember-resolver",
   "version": "4.3.0",
   "description": "The default modules based resolver for Ember CLI.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "Robert Jackson <me@rwjblue.com>",
   "directories": {
     "test": "tests"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ember-cli/ember-resolver.git"
+  },
   "scripts": {
     "build": "ember build",
     "start": "ember server",
     "test": "ember try:each"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ember-cli/ember-resolver.git"
-  },
-  "engines": {
-    "node": ">= 4"
+  "dependencies": {
+    "@glimmer/resolver": "^0.4.1",
+    "babel-plugin-debug-macros": "^0.1.10",
+    "broccoli-funnel": "^1.1.0",
+    "broccoli-merge-trees": "^2.0.0",
+    "ember-cli-babel": "^6.3.0",
+    "ember-cli-shims": "^1.1.0",
```

Comment:
Yeah this is the main thrust here. I should split them out but was too lazy to rebase w/ lockfile :-p I'll split the updates out.

Basically with the new modules, you would expect that you can now *stop* using ember-cli-shims. Why would you need both the module API provided by ember/ember-cli-babel and the legacy shims as well?

However once you drop ember-cli-shims as an app dependency, nothing can `import Ember from 'ember'`. So any addons that don't have `ember-cli-shims` as a dependency fail to have that import.

There are a few ways to fix this. Open to another idea.

I'm not clear on what the best practices for addons are in the new modules world. You would expect addon authors to want to write addons with the new APIs, but obviously continue to support older versions of Ember.